packages/permissions/src/utils/has-permissions.ts

```diff
@@ -175,27 +175,13 @@ export const hasSomePermissions = (
     hasPermission(permission, actualPermissions)
   );
 
-// Returns an Array<String> of unconfigured (not passed as `actualPermissions`) permissions.
-// The shapes of the arguments are:
-// - demandedPermissions:
-//     ['ViewProducts', 'ManageOrders']
-// - actualPermissions:
-//     { canViewProducts: true, canManageOrders: false }
-export const getInvalidPermissions = (
-  demandedPermissions: TPermissionName[],
-  actualPermissions: TPermissions
-) => {
-  // All demanded permissions need to be present as an actual permission.
-  return demandedPermissions.filter((demandedPermission: TPermissionName) =>
-    isNil(actualPermissions[toCanCase(demandedPermission)])
-  );
-};
-
 const hasDemandedDataFence = (options: {
   actualDataFence: TActualDataFence;
   demandedDataFence: TDemandedDataFence;
-  selectDataFenceData: TSelectDataFenceData;
+  selectDataFenceData?: TSelectDataFenceData;
 }): boolean => {
+  if (!options.selectDataFenceData) return false;
```

Comment:
Would you log in this case too?

Reply:
we log already in the Component level (Authorized and useIsAuthorized).